### ISSUE TITLE
Update all examples to use API

### ISSUE THREAD

**Issue by jeremyje** [googleforgames/open-match#176]
Currently there are many examples in Open Match that directly access the database instead of going through an API. This is problematic as the {MMF, Director, Evaluator} need to understand the schema of Open Match. This effectively makes the schema an API which make it difficult in the future to change without breaking customers.

 * We should remove or rewrite the -manual examples to use the APIs available.
 * Add APIs necessary to unblock common use cases.

**Comment by jeremyje** [comment]
The first step is to rewrite clientloadgen which populates player records to use the frontendapi.

**Comment by joeholley** [comment]
I think you might get more return on eng time investment using your previous proposal of using an existing load testing harness like Locust to run headless clients based on the code in frontendclient.  The CLG was never meant to speak to the FE so there would be a lot of retrofitting required that will replicate work already done in the frontend API and frontendclient.

**Comment by sawagh** [comment]
No examples currently use Redis directly.
